pkg/flowaggregator/preprocessor.go

```diff
@@ -100,7 +100,7 @@ func makeDefaultElementsWithValue(infoElements []*entities.InfoElement) ([]entit
 	return elementsWithValue, nil
 }
 
-func newPreprocessor(infoElementsV4, infoElementsV6 []*entities.InfoElement, inCh <-chan *entities.Message, outCh chan<- entities.Record) (*preprocessor, error) {
+func newPreprocessor(infoElementsV4, infoElementsV6 []*entities.InfoElement, inCh <-chan *entities.Message, outCh chan<- *entities.Message) (*preprocessor, error) {
```

Comment:
@tnqn I had to revert from a Record output channel back to a Message output channel. I wanted the ability to preserve message header information (observation domain id, exporter address) so that the information can be added to proxied records using the standard `originalX` information elements (e.g. `originalObservationDomainId`).

Reply:
got it